lib/actions/request_review.js

```diff
@@ -1,10 +1,27 @@
 const { Action } = require('./action')
+const logger = require('../logger')
 const _ = require('lodash')
 
 const createRequestReview = async (context, number, reviewers) => {
-  return context.octokit.pulls.requestReviewers(
-    context.repo({ pull_number: number, reviewers })
-  )
+  let res
+  try {
+    res = await context.octokit.pulls.requestReviewers(
+      context.repo({ pull_number: number, reviewers })
+    )
+  } catch (err) {
+    // skip when error is "Review can not be requested from pull request author"
+    if (!err.message.toLowerCase().includes('pull request author')) {
+      const errorLog = {
+        log_type: logger.logTypes.REQUEST_REVIEW_FAIL_ERROR,
+        eventId: context.eventId,
+        repo: context.payload.repository.full_name,
+        action_name: this.name
+      }
+
+      this.log.info(JSON.stringify(errorLog))
```

Comment:
If we filter out the author of the Pr from the reviewers list as per recommended on L9, then this should probably be a `log.error` since it's an unknown. and unexpected error.